src/Microsoft.ML.Data/DataLoadSave/DataReaderExtensions.cs

```diff
@@ -10,14 +10,12 @@ namespace Microsoft.ML
 {
     public static class DataReaderExtensions
     {
-        public static IDataView Read(this IDataReader<IMultiStreamSource> reader, string path)
```

Comment:
>string path [](start = 82, length = 11)

[https://stackoverflow.com/questions/38679180/will-params-in-c-sharp-always-cause-a-new-array-to-be-allocated-on-every-call](https://stackoverflow.com/questions/38679180/will-params-in-c-sharp-always-cause-a-new-array-to-be-allocated-on-every-call)
Can you please remain this method?

Reply:
Ok I understand your point, was not aware of that! That's a good thing to know :)

---
In reply to: [254525235](https://github.com/dotnet/machinelearning/pull/2417#discussion_r254525235) [](ancestors = 254525235)

Reply:
I have the impression that in many places in our code we simply have one overload with `params`.

---
In reply to: [255284570](https://github.com/dotnet/machinelearning/pull/2417#discussion_r255284570) [](ancestors = 255284570,254525235)

Reply:
Are you sure? If we talk about transforms we usually have simple case with input/output pair which is usually enough

---
In reply to: [255284582](https://github.com/dotnet/machinelearning/pull/2417#discussion_r255284582) [](ancestors = 255284582,255284570,254525235)